### DISCUSSION TITLE
How to download a file created on a remote jupyter notebook?

### DISCUSSION THREAD

**Discussion by fungiboletus** [microsoft/vscode-jupyter#11120] [Questions and Answers]
Hi,

I have a remote jupyter notebook that creates a file. Is there an easy way in vscode to download the file?

**Comment by DonJayamanne** [comment]
hmm, good question. let me log into this. at this stage i don't think there's a way to do this in vscode, 
probably the way to get this is via the remote file system explorer mentioned here https://github.com/microsoft/vscode-jupyter/issues/1366
